akka/src/main/scala/anorm/IOResult.scala

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package anorm
+
+import scala.util.{ Failure, Try }
+
+/**
+ * Holds a result of an IO operation.
+ *
+ * @param count Numeric value depending on context, for example IO operations performed or bytes processed.
+ * @param status Status of the result. Can be either [[akka.Done]] or an exception.
+ */
+final case class IOResult private[anorm] (count: Long, status: Try[Done]) {
```

Comment:
Any way, the Akka [`IOResult`](http://doc.akka.io/api/akka/2.4.8/?_ga=1.256176713.331620270.1455976396#akka.stream.IOResult) can be used